1.8/s2i/bin/usage

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DISTRO=`cat /etc/*-release | grep ^ID= | grep -Po '".*?"' | tr -d '"'`
```

Comment:
This don't work in Fedora... but I think it is not big problem.

Reply:
Not sure, I have borrowed that from sclorg s2i-python-containers.

Reply:
We'll want to have a Fedora based golang image at some point, so I'd rather say let's fix this so ti works in Fedora as well now already, before we forget that it is broken.